src/Sylius/Bundle/ApiBundle/DataTransformer/OrderItemIdAwareInputCommandDataTransformer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DataTransformer;
+
+use Sylius\Bundle\ApiBundle\Command\OrderItemIdAwareInterface;
+
+class OrderItemIdAwareInputCommandDataTransformer implements CommandDataTransformerInterface
+{
+    public function transform($object, string $to, array $context = [])
+    {
+        $explodedUri = explode('/', $context['request_uri']);
```

Comment:
Can we fetch from the context that this is really our endpoint with `orderItemId` at the end?

Reply:
This Transformer is longer exist :D thanks to your comment I changed it to subresource one and it works fine